reV/handlers/outputs.py

```diff
@@ -119,15 +119,14 @@ def writable(self):
 
         Returns
         -------
-        bool
+        writable : bool
             Flag if mode is writable
         """
+        writable = True
         mode = ['a', 'w', 'w-', 'x']
         if self._mode not in mode:
-            raise HandlerRuntimeError('mode must be writable: {}'
```

Comment:
@MRossol, note that I removed this exception. Not sure why we raised an error here.

Reply:
No idea, probably changed the usage along the way and forgot to remove...